packages/nfts/contracts/trailblazers-season-2/BadgeMigration.sol

```diff
@@ -0,0 +1,606 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.24;
+
+import "@openzeppelin/contracts-upgradeable/token/ERC1155/extensions/ERC1155SupplyUpgradeable.sol";
```

Comment:
please add new contracts into [gen-layouts.sh](https://github.com/taikoxyz/taiko-mono/blob/main/packages/nfts/deployments/gen-layouts.sh) so we always update the [layout file](https://github.com/taikoxyz/taiko-mono/blob/main/packages/nfts/contract_layout.md) for comparison.

One example: https://github.com/taikoxyz/taiko-mono/pull/18276/files